versatica/mediasoup-client
# Persist changes in sending encodings

Originally done by @thebongy here: https://github.com/versatica/mediasoup-client/pull/245

Cosmetic changes done and more handlers included.

### Steps to verify it

- Open mediasoup-demo.
- Enable mic and webcam (VP8 so with simulcast).
- In local view, enable "controls" and decrease max spatial layer.
- In console run `CC.disableMic()`.
- Check sending webcam spatial layers. It should be the same as before.
